File: backend/src/main/java/top/vuhe/entity/log/UserLogDao.java
package top.vuhe.entity.log;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * @author zhuhe
 */
@Data
@EqualsAndHashCode(callSuper = true)
@TableName(value = "user_log")
public class UserLogDao extends UserLog {
    private Integer userId;
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/entity/log/UserLogDAO.kt
package top.vuhe.sw.entity.log

import com.baomidou.mybatisplus.annotation.TableName
import java.util.*

@TableName(value = "user_log")
data class UserLogDAO(
    val id: Int?,
    val userId: Int?,
    val changeTime: Date?,
    val operationDetail: String?
):UserLog
